<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageSystemVolumes</class>
 <widget class="QDialog" name="manageSystemVolumes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage System Volumes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pbDone">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Done</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbFile">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Fi&amp;le</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPartition">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Dev&amp;ice</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <column>
    <property name="text">
     <string>Path To System Volumes</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
